spec/support/sorting_helpers.rb

```diff
@@ -1,5 +1,5 @@
 def click_header(text)
-  find(:xpath, "//table[contains(@class, 'js-sticky')]/thead/tr/th[@data-column='#{text}']").click
+  find(:xpath, "//table[contains(@class, 'sortable')]/thead/tr/th[@data-column='#{text}']").click
```

Comment:
Not super sure what's up here. There is a js-sticky table dynamically created that is prepended in DOM before the sortable table. The old tests were looking to it, but pointing the tests to the sortable table made them pass.